spec/lib/orcid/pub_identifier_mapper_spec.rb

```diff
@@ -246,6 +258,12 @@
 
       it 'ignores self identifier' do
```

Comment:
is this test title still accurate?

Reply:
Accurate, but not fully descriptive. Fixed.